pkg/api/healthcheck.go

```diff
@@ -0,0 +1,71 @@
+package api
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	lambdaTypes "github.com/aws/aws-sdk-go-v2/service/lambda/types"
+
+	"github.com/aws/aws-sdk-go-v2/service/lambda"
+	"github.com/common-fate/apikit/apio"
+	"github.com/common-fate/apikit/logger"
+	"github.com/common-fate/common-fate/pkg/cfaws"
+	"github.com/common-fate/common-fate/pkg/deploy"
+	"github.com/common-fate/common-fate/pkg/storage"
+)
+
+func (a *API) AdminRunHealthcheck(w http.ResponseWriter, r *http.Request) {
+	ctx := context.Background()
+
+	log := logger.Get(ctx)
+	dc, err := deploy.LoadConfig(deploy.DefaultFilename)
```

Comment:
General feedback, You cannot use deploy.LoadConfig in this context because the deployment.yml file is not available in the lambda runtime.